src/plugin.py

```diff
@@ -398,11 +398,18 @@ async def get_owned_games(self, owned_title_ids=None, online_check_success=False
             game = self.create_game_from_title_id(title_id)
             if game not in self.owned_games_cache:
                 log.debug("ROCKSTAR_ADD_GAME: Adding " + title_id + " to owned games cache...")
-                self.add_game(game)
                 self.owned_games_cache.append(game)
 
         return self.owned_games_cache
 
+    if IS_WINDOWS:
+        async def get_local_size(self, game_id: str, context: Any) -> Optional[int]:
+            title_id = get_game_title_id_from_ros_title_id(game_id)
+            game_status = self.check_game_status(title_id)
+            if game_status.local_game_state == LocalGameState.None_:
+                return 0
```

Comment:
API has changed its docs recently about this case: https://github.com/gogcom/galaxy-integrations-python-api/commit/f82cab2770a076328a5832b374daec02182b5776. I think you can skip this check - Galaxy should ask only for games reported as installed from `get_local_games`

Reply:
I just removed the check for installation status here in #122. Thanks for letting me know.